README.md

```diff
@@ -127,13 +127,73 @@ First-Party Sets is a proposal to standardize a mechanism that solves this issue
 
 # Applications
 
-In support of the various browser privacy models, first-party sets only control when embedded content that would otherwise be considered third-party can access its own state. Examples:
+Various applications of First-Party Sets have been suggested by different parties. The list below is an effort to compile suggested applications. 
+
+## Initial Proposed Applications
+
+**Bounce/navigation-based tracking protections**\
+_Interest from: [Safari](https://github.com/privacycg/first-party-sets/issues/53#issuecomment-902396090), Chrome, Edge_
+
+To allow for first-party cross-site single-sign-on flows to continue working, where [navigation-based tracking mitigations](https://privacycg.github.io/nav-tracking-mitigations/) are applied by browsers. Sign-on typically involves exchange of authentication tokens or other user identifiers via URL parameters on top-level navigation and redirects; which may get classified as navigational tracking.
+
+**Relaxing full third-party cookie blocking for cookies with the SameParty attribute**\
+_Interest from: Chrome, Edge_
+
+To help websites maintain various uses of cross-site cookies within the same organization such as for SSO, consent management, sandboxed domains, etc. 
+Cookies need to be labeled with the [SameParty attribute](#sameparty-cookies-and-first-party-sets).
+
+Prior art: Firefox and Edge use the [entity list](https://github.com/mozilla-services/shavar-prod-lists#entity-list), Safari’s [Quirks.cpp](https://webkit-search.igalia.com/webkit/rev/0be50a7e5c29546e4c5f27ee6b808f9aaadb9f80/Source/WebCore/page/Quirks.cpp#1154) and [affiliated domains](https://www.w3.org/2017/11/06-webappsec-minutes.html#item12) proposal.
+
+**Cookie partitioning key**\
+_Interest from: Chrome, Edge_
+
+To allow third-party software-as-a-service vendors \([reference use-case](https://github.com/privacycg/first-party-sets/issues/33)\) to continue supporting websites whose user flows span multiple sites/domains via cookies. Third-party sites must opt-in via the Partitioned attribute to partition the cookie by the [top-level website’s FPS](https://github.com/WICG/CHIPS#chips-and-first-party-sets).
+
+## Other potential applications under discussion
+**Storage partitioning key**\
+_Under consideration by: [Chrome](https://github.com/privacycg/first-party-sets/issues/37), Edge_
+
+To allow third-party software-as-a-service vendors to continue supporting websites whose user flows span multiple sites/domains via storage \(IndexedDB, localStorage, sessionStorage, Service Workers, etc.\); [storage partitioning](https://github.com/wanderview/quota-storage-partitioning/blob/main/explainer.md) could use the top-level website’s FPS \(instead of top-level site\) as a partition key. However, there are security implications \(TODO: need to write up security considerations\) that need to be accounted for before making such a change.
+
+**Cache partitioning key**\
```

Comment:
Same as above.  Isn't this all storage?

Reply:
See Erik’s summary [here](https://github.com/privacycg/first-party-sets/issues/35#issuecomment-797064339) from a previous discussion on this topic that led to a different strategy than for cookies. Essentially, there are side-channel concerns with cache or service workers sharing the same partition across domains in a FPS, some of which may host untrusted (e.g. user-uploaded content) content; so we’d like to consider a per-domain opt-in to ensure that web developers understand where there are XS-leak implications.